data/reference/syntax.adoc

```diff
@@ -137,138 +164,88 @@ literal
 open
 --
 
-[pass]
-++++
-passthrough
-++++
-
-[quote,attribution,citation title and information]
+[,attribution,citetitle]
 ____
 quote
 ____
 
-[sidebar]
+[verse,attribution,citetitle]
+____
+verse
+____
+
 ****
 sidebar
 ****
 
-[source,language,options]
-----
-source
-----
+++++
+pass
+++++
 
-// activate stem support using :stem: attribute.
+// activate stem support using `:stem:` attribute
 [stem]
 ++++
 x = y^2
 ++++
 
-
-[verse]
-____
-verse
-____
-
 == Tables
 
-.An example table
-[%header,cols=^;2m,width=40%,frame=topbot]
-|===
-|Column 1 |Column 2
-|1        |Item 1
-|2        |Item 2
-|3        |Item 3
-|6        |Three items
+.Moto Parts
+[cols=^;3e,frame=topbot,width=50%]
 |===
+| ID | Part Name
 
-////
-Common attributes:
-
-grid:    none, cols, rows, all
-frame:   topbot, none, sides, all
-options: header, footer
-format:  psv, csv, dsv
-valign:  top, bottom, middle
-width:   1%..100%
-cols:    colspec[,colspec,...]
-
-colspec:    [multiplier*][align][width][style]
-multiplier: 1...
-width:      1... or 1%...100%
-align:      [horiz][.vert]
-            horiz: < (left), ^ (center), > (right)
-            vert:  < (top),  ^ (middle), > (bottom)
-style:      d[efault], e[mphasis], m[onospaced], a[sciidoc],
-            s[trong], l[iteral], v[erse], h[eader]
-cell:       [cellspec]|data
-cellspec:   [span*|+][align][style]
-span:       [colspan][.rowspan]
-            colspan: 1...
-            rowspan: 1...
-////
-
-== Admonition
-
-NOTE: Please note that...
+| A12
+| Front Fork
 
-TIP: Pro tip...
+| B02
+| Gear Shifter
 
-IMPORTANT: Don't forget...
-
-WARNING: Watch out for...
-
-CAUTION: Ensure that...
+| C22
+| Pneumatic Tires
+|===
 
-== Images
+.Table Attributes
+[%autowidth,cols=>h;d]
+|===
+| options
+| header, footer, autowidth
 
-image::path/to/file.jpg[caption,width,height]
+| cols
+| colspec[;colspec;...]
 
-inline image:path/to/file.png[caption,title="title"].
+| grid
+| all, cols, rows, none
 
-== Videos
+| frame
+| all, sides, topbot, none
 
-video::path/to/file.mp4[width=640,start=60,end=140,options=autoplay]
+| stripes
+| all, even, odd, none
 
-video::rPQoq7ThGAU[youtube]
+| width
+| (0%..100%)
 
-video::67480300[vimeo]
+| format
+| psv, csv, dsv
+|===
 
-== Stem
+== Multimedia
 
-inline stem:[sqrt(4) = 2]
```

Comment:
No inline stem 😞

Reply:
I think I'm going to add a new group under text formatting that covers both stem and pass.